schedule-pod-restart/README.md

```diff
@@ -0,0 +1,48 @@
+# Cronjob Pod Restart
+
+Automatically restarting pods with labels based on a cron schedule.
+
+## TL;DR
+
+```console
+helm repo add coditeu https://coditeu.github.io/charts
+helm repo update
+helm install coditeu/cronjob-pod-restart
+```
+
+## Introduction
+
+## Installing the Chart
+
+To install the chart with the release name `schedule-pod-restart`:
+
+```console
+$ helm install schedule-pod-restart coditeu/schedule-pod-restart
```

Comment:
Maybe we should give a more representable example where schedule & labels are added?

For example:
```suggestion
$ helm install schedule-pod-restart coditeu/schedule-pod-restart --set "cronjob.schedule=0 1 * * *" --set cronjob.podLabels=app=example
```

Although I haven't tried it so please make sure to test this first 😅